hotspot/build.gradle

```diff
@@ -0,0 +1,18 @@
+plugins {
+    id 'io.deephaven.project.register'
+}
+
+configurations {
+    compile.extendsFrom configurations.dhUtil
+}
+
+dependencies {
+    compile project(':log-factory')
+
+    testRuntime project(path: ':configs'), project(path: ':test-configs')
+
+    testRuntimeOnly project(':log-to-slf4j')
+    Classpaths.inheritSlf4j(project, 'slf4j-simple', 'testRuntimeOnly')
+
+    Classpaths.inheritJUnitPlatform(project)
+}
```

Comment:
We should not use compile or testRuntime dependencies for new projects. I don't think these dependencies are necessary?